docs/statements.md

```diff
@@ -290,6 +291,58 @@ A typical usage pattern for this class might be:
 
 With the above pattern the transaction is committed only when the code successfully reaches the end of block. If some exception is thrown before that, the scope will be left without reaching the final statement and the transaction object will automatically roll back in its destructor.
 
+### <a name="ddl"></a> Portable DDL
+
+SOCI supports some basic methods to construct portable DDL queries. That is, instead of writing explicit SQL statement for creating or modifying tables, it is possible to use dedicated SOCI functions, which prepare appropriate DDL statements behind the scenes, thus enabling the user application to create basic database structures in a way that is portable across different database servers. Note that the actual support for these functions depends on the actual backend implementation.
+
+It is possible to create a new table in a single statement:
+
+   sql.create_table("t1").column("i", soci::dt_integer).column("j", soci::dt_integer);
+
+Above, table "t1" will be created with two columns ("i", "j") of type integer.
+
+It is also possible to build such a statement piece by piece:
```

Comment:
Just to avoid any confusion, I'd make it more clear that this piece-by-piece example is not exactly the same as the one liner above.